README.md

```diff
@@ -96,3 +96,61 @@ running app.
 ```sh
 # 2015/03/13 18:22:43 Starting up HTTP server at :9197.
 ```
+
+## Testing autoupdates locally
+
+Use the `mock` build tag to build a binary that mocks Github's API responses.
+
+```
+go build -tags mock github.com/getlantern/autoupdate-server
+```
+
+The you can run this binary using the publicly available private key for
+testing:
+
+```
+autoupdate-server \
+	-k _resources/example-keys/private.key \
+	-l 127.0.0.1:9999 \
+	-p http://127.0.0.1:9999/ \
+	-o getlantern \
+	-n lantern
+```
+
+Alternatively, you could use the `Makefile` script that automatizes this task by
+using a docker container:
+
+```
+make mock-server
+```
+
+A mock server will be available at `127.0.0.1:9999`, you can point Lantern to
+that server by editing the `updateserverurl` property on the
+`lantern-x.y.z.yaml` file:
```

Comment:
We split the yaml to two files recently, `global.yaml` and `proxies.yaml` respectively, both encrypted. The `updateserverurl` field doesn't have effect too.

We probably can stick with the single method at L152 to test against mock autoupdate server, and remove all other methods.